src/test/ui/argument-suggestions/basic.stderr

```diff
@@ -0,0 +1,92 @@
+error[E0308]: arguments to this function are incorrect
+  --> $DIR/basic.rs:20:5
+   |
+LL |     invalid(1.0);
+   |     ^^^^^^^
+   |
+note: expected `u32`, found floating-point number
+  --> $DIR/basic.rs:20:13
+   |
+LL |     invalid(1.0);
+   |             ^^^
+note: function defined here
+  --> $DIR/basic.rs:13:4
+   |
+LL | fn invalid(_i: u32) {}
+   |    ^^^^^^^
+
+error[E0061]: this function takes 0 arguments but 1 argument was supplied
+  --> $DIR/basic.rs:21:5
+   |
+LL |     extra("");
+   |     ^^^^^ -- argument unexpected
```

Comment:
This should be

```suggestion
   |     ----- ^^ unexpected argument
```

Reply:
Is there any way to do this other than changing the primary span of the error and the span of the label? (Though, then the message would be different...)